### ISSUE TITLE
Support ModuleNotFoundError (in Python 3.6)

### ISSUE THREAD

**Issue by frexvahi** [python/mypy#3252]
```python
try:
    import foo
except ModuleNotFoundError:
    pass
````
mypy 0.501 on python 3.6 says `error: Name 'ModuleNotFoundError' is not defined`

I also had problems with this (valid for Python>=3.3 according to https://docs.python.org/3/library/exceptions.html)
```python
try:
    import foo
except ImportError as exc:
    if exc.name == 'bar':  # importing 'foo' failed because dependency 'bar' is not installed
        pass
    else:
        raise
```
mypy says `error: "ImportError" has no attribute "name"`

**Comment by JelleZijlstra** [comment]
Both of these sound like issues with http://github.com/python/typeshed.

I can't reproduce your problem with ModuleNotFoundError: it's there in https://github.com/python/typeshed/blob/master/stdlib/3/builtins.pyi#L915 and on mypy master using ModuleNotFoundError in 3.6 mode works fine.

The `name` and `path` attributes are indeed missing from ImportError in typeshed. This should be easy to fix; feel free to open a PR to typeshed.

**Comment by frexvahi** [comment]
I tried with mypy installed from github master, the first issue has indeed been fixed since the 0.501 release.

Since the ImportError problem comes from typeshed, I am closing this issue.
